hknweb/events/views.py

```diff
@@ -62,10 +71,8 @@ def rsvp(request, id):
     event = get_object_or_404(Event, pk=id)
     rsvps = event.rsvp_set.count()
 
-    if request.user.is_authenticated and (event.rsvp_limit is None or rsvps < event.rsvp_limit):
```

Comment:
To clarify, we didn't need to check user is authenticated in the first place because the decorator did that right?